src/sentry/static/sentry/app/views/settings/components/settingsLayout.jsx

```diff
@@ -27,21 +27,24 @@ class SettingsLayout extends React.Component {
       <React.Fragment>
         <SettingsColumn>
           <SettingsHeader>
-            <Flex align="center" width={1}>
-              <Box flex="1">
-                <SettingsBreadcrumb
-                  params={params}
-                  routes={childRoutes}
-                  route={childRoute}
-                />
-              </Box>
-              <SettingsSearch routes={routes} router={router} params={params} />
-            </Flex>
+            <StyledBreadcrumbWrapper>
+              <SettingsHamburgerIcon
```

Comment:
You can make this `priority="link"` or something along those lines to get rid of the button styles.